src/test/java/com/wikia/webdriver/common/core/elemnt/JavascriptActions.java

```diff
@@ -1,5 +1,6 @@
 package com.wikia.webdriver.common.core.elemnt;
 
+import com.google.common.base.Optional;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.google.common.base.Optional'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUselessImportCheck)

Reply:
Let's remove it if it's really unused ;)